lib/content-services/src/lib/search/components/search-facet-field/search-facet-field.component.spec.ts

```diff
@@ -62,7 +62,7 @@ describe('SearchFacetFieldComponent', () => {
             component.onToggleBucket(event, field, bucket);
 
             expect(bucket.checked).toBeTruthy();
-            expect(queryBuilder.addUserFacetBucket).toHaveBeenCalledWith(field, bucket);
+            expect(queryBuilder.addUserFacetBucket).toHaveBeenCalledWith(field.field, bucket);
```

Comment:
Can we maybe rename the unit test variable with something else? field.field sounds weird

Reply:
Renamed